config/config.go

```diff
@@ -1488,9 +1489,9 @@ type QueueConfig struct {
 
 // RemoteReadConfig is the configuration for reading from remote storage.
 type RemoteReadConfig struct {
-	URL           *URL           `yaml:"url"`
-	RemoteTimeout model.Duration `yaml:"remote_timeout,omitempty"`
-
+	URL             *URL           `yaml:"url"`
+	RemoteTimeout   model.Duration `yaml:"remote_timeout,omitempty"`
+	IsRemoteStorage bool           `yaml:"is_remote_storage,omitempty"`
```

Comment:
I'm not sure this is the right approach for this, I'm expecting a few different settings depending on what exactly is on the other end. I think a setting that specifically addresses this behaviour along the lines of `ReadRecent` might be better.

Reply:
Done